docs/static/gamejam/jams/cozy/rules.md

```diff
@@ -1,59 +1,27 @@
-## Cozy Jam
+## Cozy Jam Winners!
 
-The theme for this jam is "cozy games". What is a [cozy game](https://en.wikipedia.org/wiki/Cozy_game), you might ask? Well, it's any game that focuses on creating a relaxing atmosphere rather than a stressful one.
-Cozy games are often about building things, exploring the world, or expressing yourself instead of limited lives, time limits, or difficulty.
-Cozy games can also be a game that just has a real cozy vibe! Think warm mugs of hot cocoa, friendly conversations with neighbors, soft pastel colors... you get the idea!
+## First Place: Rhythm Code!
+| |
+| -- |
+| [!["Rhythm Code!" game screenshot](https://cdn.makecode.com/api/40921-79622-15624-16091/thumb)](https://arcade.makecode.com/40921-79622-15624-16091) |
+| &nbsp; |
 
+[Rhythm Code!](https://arcade.makecode.com/40921-79622-15624-16091) by **InvalidProject** features not just one, but THREE whole rhythm games to test your timing skills! Our judges loved the artwork, music, and rating system of this submission. Plus, rumor has it that each of these games is based on a different member of the MakeCode forums... Make sure you check out [this forum post](https://forum.makecode.com/t/my-game-jam-game-rhythm-code/32951) for more detailed instructions on how to play each of the games!
```

Comment:
I know the fourth is just a combination of all the three others, but do we want to add a detail about that one, too?